services/contract_call_local.proto

```diff
@@ -58,6 +58,23 @@ message ContractLoginfo {
     bytes data = 4;
 }
 
+/**
+ * Info about a contract account's nonce value.
+ * A nonce of a contract is only incremented when that contract creates another contract.
+ */
+message ContractNonceInfo {
+
+    /**
+    * Id of the contract
+    */
```

Comment:
Indentation on these comments (and other new comments) are off.

Reply:
Fixed.